.gitignore

```diff
@@ -27,7 +27,6 @@ build/Release
 
 # Dependency directory
 node_modules
-typings
```

Comment:
isn't this old / left over from the old typings from the old version of typescript?